### ISSUE TITLE
ES Module not being shipped

### ISSUE THREAD

**Issue by timwis** [Leaflet/Leaflet#6131]
## How to reproduce

- Leaflet version I'm using: 1.3.1
- Browser (with version) I'm using: N/A
- OS/Platform (with version) I'm using: macOS
- `npm install leaflet`

## What behaviour I'm expecting and which behaviour I'm seeing
Per the source code's [package.json](https://github.com/Leaflet/Leaflet/blob/master/package.json#L34), I expect `dist/leaflet-src.esm.js` to exist and be referenced by the `module` field, so webpack can optimise my bundle size and only import what I need. Instead, the file does not exist in `node_modules/leaflet`, and there's no reference to it in the imported package.json. The imported package.json does say version `1.3.1` though.

## Minimal example reproducing the issue
<img width="188" alt="screen shot 2018-04-10 at 6 15 09 am" src="https://user-images.githubusercontent.com/761444/38551247-b6578178-3c86-11e8-9988-a11b85f576ae.png">

**Comment by cherniavskii** [comment]
Maybe I'll have some time to release it next week

**Comment by piuccio** [comment]
@cherniavskii I think this can be closed, ESM was included in `1.3.2`

```
> ls -1 node_modules/leaflet/dist/
images
leaflet-src.esm.js
leaflet-src.esm.js.map
leaflet-src.js
leaflet-src.js.map
leaflet.css
leaflet.js
leaflet.js.map
```
